### PR TITLE
gdalbuildvrt: add page

### PR DESCRIPTION
**Pull request by RobinKohrs**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by navarroaxel** [comment]
@RobinKohrs Could you please sign the license/CLA to pass the check?

**Comment by RobinKohrs** [comment]
@navarroaxel Sure thing! sorry about that. I'm completely new to this and just wanted the help on the command line. Thanks for pointing out to that.

File: pages/common/gdalbuildvrt.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+> Build Virtual Datasets from a list of existing datasets.
+> More information: <https://gdal.org/programs/gdalbuildvrt.html>.
+
+- Make a virtual mosaic from all TIFF files contained in a directory:
+
+`gdalbuildvrt {{ output.vrt }} {{ input_folder/*.tif }}`
+
+- Make a virtual mosaic from files whose name is specified in a text file:
+
+`gdalbuildvrt -input_file_list {{ my_list.txt }} {{ output.vrt }}`
```
Comment (navarroaxel) [inline pages/common/gdalbuildvrt.md:12]:
```suggestion
`gdalbuildvrt -input_file_list {{path/to/list.txt}} {{path/to/output.vrt}}`
```

File: pages/common/gdalbuildvrt.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# gdalbuildvrt
+
+> Build Virtual Datasets from a list of existing datasets.
+> More information: <https://gdal.org/programs/gdalbuildvrt.html>.
+
+- Make a virtual mosaic from all TIFF files contained in a directory:
+
+`gdalbuildvrt {{ output.vrt }} {{ input_folder/*.tif }}`
```
Comment (navarroaxel) [inline pages/common/gdalbuildvrt.md:8]:
```suggestion
`gdalbuildvrt {{path/to/output.vrt}} {{path/to/input_directory/*.tif}}`
```

File: pages/common/gdalbuildvrt.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Make a virtual mosaic from files whose name is specified in a text file:
+
+`gdalbuildvrt -input_file_list {{ my_list.txt }} {{ output.vrt }}`
+
+- Make a RGB virtual mosaic from 3 single-band input files:
+
+`gdalbuildvrt -separate {{ rgb.vrt }}  {{ red.tif green.tif blue.tif }}`
```
Comment (navarroaxel) [inline pages/common/gdalbuildvrt.md:16]:
```suggestion
`gdalbuildvrt -separate {{path/to/rgb.vrt}}  {{path/to/red.tif}] {{path/to/green.tif}} {{path/to/blue.tif}}`
```

File: pages/common/gdalbuildvrt.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Make a virtual mosaic with blue background colour (RGB: 0 0 255):
+
+`gdalbuildvrt -hidenodata -vrtnodata "0 0 255" {{ output.vrt }} {{ input_folder/*.tif }}`
```
Comment (navarroaxel) [inline pages/common/gdalbuildvrt.md:20]:
```suggestion
`gdalbuildvrt -hidenodata -vrtnodata "{{0 0 255}}" {{path/to/output.vrt}} {{path/to/input_directory/*.tif}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @RobinKohrs! Looks ok to me. Welcome to tldr pages :smiley_cat:

**Comment by RobinKohrs** [comment]
Thanks so much!:) I'm relatively new to this and didn't know how exactly how to accept the changes @navarroaxel made. But glad it somehow worked.

**Comment by navarroaxel** [comment]
Don't worry @RobinKohrs You can check how to accept the suggestions here: https://docs.github.com/en/enterprise-server@2.21/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request

**Pull request merged**